app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java

```diff
@@ -0,0 +1,47 @@
+package it.quip.android.partials;
```

Comment:
Also, @jcomo couldn't make an interface package unfortunately.  Had to go with "interfaces."  Shold we really conform to java style or just accept the android style on this